### PR TITLE
update dependencies (nuget and npm)

### PR DESCRIPTION
**Pull request by 0x53A**
I haven't run any manual tests, that's what CI is for

### CODE REVIEW

**Comment by 0x53A** [comment]
FontAwesome was removed from Fulma and added to its own package: https://github.com/MangelMaxime/Fulma/pull/166#issuecomment-443319911

For now I was lazy and locked Fulma to the old version.

**Comment by theimowski** [comment]
Thanks for this! Can we add FontAwesome explicitly as well please? See #198

**Comment by 0x53A** [comment]
I have no clue how FA works.

Can you tell me what to replace the old

```f#
Icon.faIcon [ ]
  [ Fa.icon Fa.I.Github; Fa.fw ]
```

with?

**Comment by 0x53A** [comment]
@isaacabraham Do you know what the new syntax is? Never used FA.

**Comment by isaacabraham** [comment]
@0x53A It's the same syntax as before, but in new Elmish you need to add the Fable.FontAwesome.Free package as well (and open `Fable.FontAwesome` I believe). Otherwise it should be as before (except there are more icons now as it goes against FA 5, not 4).

**Comment by 0x53A** [comment]
@isaacabraham I tried it, but the new package seems to have completely different content.

grafik

**Comment by isaacabraham** [comment]
cc @MangelMaxime

**Comment by MangelMaxime** [comment]
You can check out the migration guide and new documentation:
https://mangelmaxime.github.io/Fulma/#blog-viewer?file=blog/articles/fulma_2.0.0.md

File: Content/src/Client/Client.fs [hunk 1]
```diff
@@ -520,8 +521,8 @@ let navMenu =
                       Button.IsOutlined
                       Button.Size IsSmall
                       Button.Props [ Href "https://github.com/SAFE-Stack/SAFE-template" ] ]
-                    [ Icon.faIcon [ ]
-                        [ Fa.icon Fa.I.Github; Fa.fw ]
+                    [ Icon.icon [ ]
+                        [ Fa.i [Fa.Brand.Github] [] ]
```
[the remark this answers is not in the document]
Comment (MangelMaxime) [reply Content/src/Client/Client.fs]: `Fa.fw` stand for `fiwed-width`

The replacement is `Fa.FixedWidth`

File: Content/src/Client/Client.fs [hunk 2]
```diff
@@ -359,9 +360,9 @@ let columns (model : Model) (dispatch : Msg -> unit) =
                                   [ for _ in 1..10 ->
                                       tr [ ]
                                           [ td [ Style [ Width "5%" ] ]
-                                              [ Icon.faIcon
+                                              [ Icon.icon
                                                   [ ]
-                                                  [ Fa.icon Fa.I.BellO ] ]
+                                                  [ Fa.i [ Fa.Solid.Bell ] [] ] ]
```
[the remark this answers is not in the document]
Comment (MangelMaxime) [reply Content/src/Client/Client.fs]: `BellO` replacement is `Fa.Regular.Bell`

You can see the list of all the icon name change between version 4 and 5 [here](https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4)

File: Content/src/Client/Client.fs [hunk 3]
```diff
@@ -561,9 +561,9 @@ let card icon heading body =
...
-          card Fa.I.IdCardO "Tempor orci dapibus ultrices in." "Ut venenatis tellus in metus vulputate. Amet consectetur adipiscing elit pellentesque. Sed arcu non odio euismod lacinia at quis risus. Faucibus turpis in eu mi bibendum neque egestas cmonsu songue. Phasellus vestibulum lorem sed risus."
-          card Fa.I.Rocket "Leo integer malesuada nunc vel risus." "Imperdiet dui accumsan sit amet nulla facilisi morbi. Fusce ut placerat orci nulla pellentesque dignissim enim. Libero id faucibus nisl tincidunt eget nullam. Commodo viverra maecenas accumsan lacus vel facilisis." ]
+        [ card Fa.Solid.Paw "Tristique senectus et netus et." "Purus semper eget duis at tellus at urna condimentum mattis. Non blandit massa enim nec. Integer enim neque volutpat ac tincidunt vitae semper quis. Accumsan tortor posuere ac ut consequat semper viverra nam."
+          card Fa.Solid.IdCard "Tempor orci dapibus ultrices in." "Ut venenatis tellus in metus vulputate. Amet consectetur adipiscing elit pellentesque. Sed arcu non odio euismod lacinia at quis risus. Faucibus turpis in eu mi bibendum neque egestas cmonsu songue. Phasellus vestibulum lorem sed risus."
```
[the remark this answers is not in the document]
Comment (MangelMaxime) [reply Content/src/Client/Client.fs]: `IdCardO` replacement is `Fa.Regular.IdCard`

You can see the list of all the icon name change between version 4 and 5 [here](https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4)

**Comment by theimowski** [comment]
Awesome, thanks!

**Pull request merged**